style/global/_typography.scss

```diff
@@ -0,0 +1,79 @@
+$base-font: 'Inconsolata', 'Courier New', Courier, monospace;
+$secondary-font: 'Prata', sans;
+$decor-font: 'Brandon Grotesque Bold', 'Roboto';
+
+body {
+  font-family: $base-font;
+  line-height: normal;
+  letter-spacing: normal;
+  color: $black;
+}
+
+.hero-section__heading {
+  font-family: $secondary-font;
+  font-size: 129px;
+  line-height: 0.7;
+  font-weight: 400;
+  color: $white;
+}
+
+.hero-section__decor {
+  font-family: $decor-font;
+  font-size: 337px;
+  line-height: 0.77;
```

Comment:
This line height and font size feel bit awkward to be honest, but I guess it is just very big.

Reply:
We can discuss this offline, as I had some problems with reading sizes. ;)